app/models/concerns/annotation_base.rb

```diff
@@ -134,7 +134,7 @@ def assign_to_users
         status_id = self.annotated&.last_status_obj&.id
         users = User.joins(:assignments).where('assignments.assigned_id' => status_id, 'assignments.assigned_type' => 'Annotation').map(&:id).uniq
       elsif self.annotation_type == 'verification_status'
-        project_id = self.annotated&.project_id
+        project_id = self.annotated&.project_ids.first
```

Comment:
Whenever you find yourself needing to replace `project_id` with `projects_ids.first`, plz try to remove this code completely instead.

Reply:
Ok